### ISSUE TITLE
curl_easy_perform() failed: SSL peer certificate or SSH remote key was not OK

### ISSUE THREAD

**Issue by yakir12** [aviks/SMTPClient.jl#37]
So like some of the others, I too got this error. This is from trying your README example (where I replaced all the you@gmail for my own and the me@test.com with another address I have access to). I feel like this example could use some more description (which I'll gladly PR if you like). For instance, you mention that all email addresses should be enclosed in angle brackets, but that's not the case in the body for the To field:
```julia
"To: me@test.com\n" *
```
In any case, here's the trace:

```julia
ERROR: LoadError: curl_easy_perform() failed: SSL peer certificate or SSH remote key was not OK
Stacktrace:
 [1] error(::String) at ./error.jl:33
 [2] macro expansion at /home/pi/.julia/packages/SMTPClient/VShKI/src/utils.jl:8 [inlined]
 [3] connect(::SMTPClient.ConnContext) at /home/pi/.julia/packages/SMTPClient/VShKI/src/types.jl:114
 [4] do_send(::String, ::Array{String,1}, ::String, ::SendOptions, ::SMTPClient.ReadData{Base.GenericIOBuffer{Array{UInt8,1}}}) at /home/pi/.julia/packages/SMTPClient/VShKI/src/mail.jl:14
 [5] send(::String, ::Array{String,1}, ::String, ::Base.GenericIOBuffer{Array{UInt8,1}}, ::SendOptions) at /home/pi/.julia/packages/SMTPClient/VShKI/src/mail.jl:2
 [6] top-level scope at /home/pi/srv/email.jl:18
 [7] include at ./boot.jl:328 [inlined]
 [8] include_relative(::Module, ::String) at ./loading.jl:1105
 [9] include(::Module, ::String) at ./Base.jl:31
 [10] include(::String) at ./client.jl:424
 [11] top-level scope at REPL[6]:1
in expression starting at /home/pi/srv/email.jl:18
```

**Comment by aviks** [comment]
Can you check what version of LibCURL.jl you have installed?

**Comment by yakir12** [comment]
I didn't `add LibCURL`, just `SMTPClient`, but now I did, and it's 'v0.5.2`

And to answer your question more accurately, here you can see that in a fresh install of `SMTPClient`, version 0.5.2 of `LibCURL` is installed:

```julia
(email) pkg> add SMTPClient
  Updating registry at `~/.julia/registries/General`
  Updating git-repo `https://github.com/JuliaRegistries/General.git`
 Resolving package versions...
  Updating `~/email/Project.toml`
  [c35d69d1] + SMTPClient v0.3.1
  Updating `~/email/Manifest.toml`
  [b99e7846] + BinaryProvider v0.5.8
  [b27032c2] + LibCURL v0.5.2
  [c35d69d1] + SMTPClient v0.3.1
  [2a0f44e3] + Base64 
  [8ba89e20] + Distributed 
  [b77e0a4c] + InteractiveUtils 
  [8f399da3] + Libdl 
  [56ddb016] + Logging 
  [d6f4376e] + Markdown 
  [9a3f8284] + Random 
  [ea8e919c] + SHA 
  [9e88b42a] + Serialization 
  [6462fe0b] + Sockets 
  [8dfed614] + Test 
```

**Comment by yakir12** [comment]
I went ahead and tried it with the following older releases of `LibCURL`: 
0.5.1
0.5.0
0.4.1
0.4.0
and it failed with exactly the same error.
I stopped at `v0.3.2` because the error changed to 
```julia
ERROR: LoadError: MethodError: no method matching curl_easy_setopt(::Ptr{Union{}}, ::Int32, ::SMTPClient.ConnContext)
```
and I figured that was enough.

**Comment by aviks** [comment]
Fixed by #39
